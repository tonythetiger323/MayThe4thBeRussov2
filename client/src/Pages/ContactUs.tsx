import * as React from 'react'
import Typography from '@material-ui/core/Typography'

class ContactUs extends React.Component {
  render() {
    return (
      <div>
        <Typography align='center' variant='h2'>
          Contact Us
        </Typography>
        <br />
        <br />
        <Typography align='center' variant='h4'>
          Questions? Use the form below to send us a message and we will get
          back to you at our ealiest convenience!
        </Typography>
      </div>
    );
  }
}

export default ContactUs;
