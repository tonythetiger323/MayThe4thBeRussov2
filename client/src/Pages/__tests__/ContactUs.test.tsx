import * as React from 'react'
import { shallow } from 'enzyme'
import ContactUs from '../ContactUs'

describe('ContactUs Page', () => {
  let wrapper: any
  beforeEach(() => (wrapper = shallow(<ContactUs />)))

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
