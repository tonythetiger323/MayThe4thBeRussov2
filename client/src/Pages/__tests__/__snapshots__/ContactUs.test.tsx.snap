// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs Page should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ContactUs />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <WithStyles(Typography)
          align="center"
          variant="h2"
        >
          Contact Us
        </WithStyles(Typography)>,
        <br />,
        <br />,
        <WithStyles(Typography)
          align="center"
          variant="h4"
        >
          Questions? Use the form below to send us a message and we will get back to you at our ealiest convenience!
        </WithStyles(Typography)>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "align": "center",
          "children": "Contact Us",
          "variant": "h2",
        },
        "ref": null,
        "rendered": "Contact Us",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "align": "center",
          "children": "Questions? Use the form below to send us a message and we will get back to you at our ealiest convenience!",
          "variant": "h4",
        },
        "ref": null,
        "rendered": "Questions? Use the form below to send us a message and we will get back to you at our ealiest convenience!",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <WithStyles(Typography)
            align="center"
            variant="h2"
          >
            Contact Us
          </WithStyles(Typography)>,
          <br />,
          <br />,
          <WithStyles(Typography)
            align="center"
            variant="h4"
          >
            Questions? Use the form below to send us a message and we will get back to you at our ealiest convenience!
          </WithStyles(Typography)>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "align": "center",
            "children": "Contact Us",
            "variant": "h2",
          },
          "ref": null,
          "rendered": "Contact Us",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "align": "center",
            "children": "Questions? Use the form below to send us a message and we will get back to you at our ealiest convenience!",
            "variant": "h4",
          },
          "ref": null,
          "rendered": "Questions? Use the form below to send us a message and we will get back to you at our ealiest convenience!",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
